src/test/kotlin/com/atlassian/performance/tools/hardware/ApplicationScale.kt

```diff
@@ -34,7 +34,7 @@ private val JIRA_XL_DATASET = StorageLocation(
                 artifactName = "database.tar.bz2",
                 location = location,
                 unpackedPath = "database",
-                downloadTimeout = Duration.ofMinutes(40)
```

Comment:
How large is the zip?

Reply:
24 GB

Reply:
You can use `HttpDatasetPackage` to download and unpack the resource faster (you would also have to use an ephemeral drive for the database). See:
https://github.com/atlassian/infrastructure/commit/57925df625ff0d7b1cb93fdcf011141bab694a3a

Reply:
That's on our radar. However, HTTP transport requires public read access - correct?
If so, we gotta sanitise the dataset first.

Reply:
> If so, we gotta sanitise the dataset first.

There's also a solution that can fit in a middle. You can use pre-signed URLs. The URLs will be hard to guess and available only temporarily. 

https://docs.aws.amazon.com/AmazonS3/latest/dev/ShareObjectPreSignedURL.html